packages/figma-scripting/plugins/_figmaToken.ts

```diff
@@ -4,4 +4,4 @@
 // and generate a new one using the "Get personal access token" link
 // or you can go directly in the Figma website, under the user settings
 // and create one from there (so you can also assign a name to it)
-export const figmaToken = '228668-d76c9279-7eca-453a-b3f6-189ed4f98b7c';
+export const figmaToken = 'copy-your-token-here';
```

Comment:
@didoo Can you please regenerate a new token for yourself? Sorry I didn't notice this until I'd already ported over the repo.

Reply:
@amyrlam OK done. 

[issue] since everyone will need to add their own personal token here, to use the scripts, we risk that someone will commit their own token again if the file is not ignored. 

[suggestion] maybe we should adopt the same approach as in the `flight` repo/package where there are two files, `.env.template` that shows how to add a token and the actual `.env` file that is ignored.

Reply:
sounds good, will work on that in this PR in a bit